lib/msal-common/src/telemetry/performance/PerformanceEvent.ts

```diff
@@ -238,6 +240,7 @@ export enum PerformanceEvents {
 
     UsernamePasswordClientAcquireToken = "usernamePasswordClientAcquireToken",
 
+    NativeMessageHandlerHandshake = "nativeMessageHandlerHandshake",
```

Comment:
This can be grouped with `NativeInteractionClientAcquireToken` above.

Reply:
Please correct me if I am wrong, but these are two completely different events... `NativeMessageHandler.sendHandshakeRequest()` does not acquire a token.

Reply:
I meant code wise, not merge. Something like this:

```
NativeInteractionClientAcquireToken = ....
NativeMessageHandler = "nativeMessageHandlerHandshake"
...
```